libs/components/popovers/src/lib/modules/dropdown/dropdown.component.html

```diff
@@ -7,7 +7,10 @@
     [attr.aria-haspopup]="menuAriaRole"
     [attr.aria-label]="
       label ||
-      (labelDefault | async) ||
+      (buttonType === 'context-menu' && contentDescriptor
```

Comment:
You should be able to factor this so that `buttonType === 'context-menu'` is only checked once (untested so caveat emptor):

```
(buttonType === 'contextMenu' 
  ? ((contentInfo | async).descriptor  
    ? ('skyux_dropdown_context_menu_with_content_descriptor_default_label' | skyLibResources : (contentInfo | async).descriptor)
     : 'skyux_dropdown_context_menu_default_label' | skyLibResources) 
  : undefined
```